### PR TITLE
update ES script to new 8.16 cluster

### PR DESCRIPTION
**Pull request by alemsh**

### CODE REVIEW

File: resources/progress_to_es.py [hunk 1]
```diff
@@ -203,10 +203,22 @@ def main():
...
+                        help='ES oauth2 client id')
+    parser.add_argument('--es_client_secret',default=None,
+                        help='ES oauth2 client secret')
+    parser.add_argument('--token_url',default=None,
+                        help='ES oauth2 realm token url')
     args = parser.parse_args()
 
     logging.basicConfig(level=getattr(logging, args.log_level.upper()), format='%(asctime)s %(levelname)s %(name)s : %(message)s')
 
+    es_api = ClientCredentialsAuth(address='https://elasticsearch.icecube.aq',
```
Comment (blinkdog) [inline resources/progress_to_es.py:216]: Maybe change this to `address=config['ES_ADDRESS']` ?

**Review by blinkdog — CHANGES_REQUESTED** [review]
Looks pretty good! Just one minor suggestion, and it should be ready to merge.

File: resources/progress_to_es.py [hunk 2]
```diff
@@ -203,10 +203,22 @@ def main():
...
+                            token_url=args.token_url,
+                            client_secret=args.es_client_secret,
+                            client_id=args.es_client_id)
```
Comment (blinkdog) [inline resources/progress_to_es.py:219]: One of the CI tests using flake8 is complaining about the under-indent here; line up the others on the same column as address.

**Review by blinkdog — CHANGES_REQUESTED** [review]
Whoops, another small change to pacify CI.

File: resources/progress_to_es.py (hunk 1 above)
Comment (alemsh) [reply resources/progress_to_es.py:216]: Sounds good

File: resources/progress_to_es.py (hunk 2 above)
Comment (alemsh) [reply resources/progress_to_es.py]: Ope, good catch updated!

**Review by blinkdog — APPROVED** [review]

**Comment by blinkdog** [comment]
LGTM. Please merge when you're ready.

**Pull request merged**